hr_expense_petty_cash_sequence/models/hr_expense_sheet.py

```diff
@@ -0,0 +1,24 @@
+# Copyright 2019 Ecosoft Co., Ltd (http://ecosoft.co.th/)
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl.html)
+
+from odoo import api, models
+
+
+class HrExpenseSheet(models.Model):
+    _inherit = 'hr.expense.sheet'
+    _rec_name = 'number'
```

Comment:
@ps-tubtim do we need to force this for petty cash? I think this line already exists in hr_expense_sequence.
Should we remove it?

Reply:
Edited.